src/locale/locales/it/messages.po

```diff
@@ -59,7 +48,7 @@ msgstr ""
 msgid "{0, plural, one {Like (# like)} other {Like (# likes)}}"
 msgstr ""
```

Comment:
If it is `mi piace` for the noun `like` as well then it would be:
```suggestion
msgstr "{0, plural, one {Mi piace (# mi piace)} other {Mi piace (# mi piace)}}"
```